contracts/nft/tasks/deploy.ts

```diff
@@ -11,6 +12,13 @@ const main = async (args: any, hre: HardhatRuntimeEnvironment) => {
     );
   }
 
+  if (
+    !isAddress(args.gateway) ||
+    (args.uniswapRouter && !isAddress(args.uniswapRouter))
```

Comment:
I wanted to use the same deploy task for both EVM and ZetaChain contracts. The only difference between them is that a universal contract on ZetaChain needs a uniswap router, so that's why it's optional.